docs/yaml.md

```diff
@@ -27,6 +27,79 @@ This stanza describes how to build the Docker image your model runs in. It conta
 
 Cog automatically picks the correct version of CUDA to install, but this lets you override it for whatever reason.
 
+### `environment`
+
+An optional list of environment variables. They will be set in the Dockerfile using [`ENV` instructions](https://docs.docker.com/engine/reference/builder/#env).
+These are set in the Docker container, so your `predict.py` and imported libraries will be able to use them.
+
+For example:
+
+```yaml
+build:
+  environment:
+    - SOME_DIR=/src/example
+    - ANOTHER_DIR=$SOME_DIR/weights
```

Comment:
Good question — yes order matters. I'll clarify that.

Regarding a realistic example, I'm with you, it's a little tough in this case because I don't want to encourage people to hardcode unnecessary variables, and `XDG_CACHE_HOME` takes care of many realistic use-cases for both PyTorch and Huggingface. I thought about using other realistic examples, but I don't want it to seem like it's endorsing a specific library, since that could get stale.

But I'll try!

Reply:
OK now it's a more realistic example, while still avoiding encouraging unnecessary vars.

![image](https://user-images.githubusercontent.com/2420688/158686317-0ec84986-b4f0-4e8c-8937-820ba51e74a9.png)